sdk/core/azure-core/CHANGELOG.md

```diff
@@ -15,6 +15,13 @@
 - Make WinHTTP transport adapter to NOT use SSL/TLS for unsecured HTTP connections.
 
 ### Other Changes
+- Updated source code to build with Clang 11. (A community contribution, courtesy of _[davidchisnall](https://github.com/davidchisnall)_)
+
+### Acknowledgments
+
+Thank you to our developer community members who helped to make Azure Core better with their contributions to this release:
+
+- David Chisnall _([GitHub](https://github.com/davidchisnall))_
```

Comment:
Thank you @davidchisnall! Your fix is now in the release - https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-core_1.2.0

Reply:
Thanks!  Will there also be releases of the other components?  I currently need a modified vcpkg port to be able to build the storage component.

Reply:
Yes, they are already released here on GitHub (https://github.com/Azure/azure-sdk-for-cpp/releases - you can see what's new in the release notes there).

They will become available in vcpkg as soon as vcpkg team completes the pull requests:
* Storage 12.1.0 - https://github.com/microsoft/vcpkg/pull/19488
* Identity 1.1.0 - https://github.com/microsoft/vcpkg/pull/19485
* KeyVault 4.0.0 - https://github.com/microsoft/vcpkg/pull/19487

Reply:
Fantastic, thank you!

Reply:
All the listed above packages that were released in early August were just made available in vcpkg.
Please let us know if something does not work for you, and if you still have to use the modified port.

Reply:
Thanks!  Since this and microsoft/vcpkg#19456 were merged, I can bootstrap vcpkg and the SDK from CMake on FreeBSD.